google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -609,6 +612,11 @@ public URL signUrl(BlobInfo blobInfo, long duration, TimeUnit unit, SignUrlOptio
     for (SignUrlOption option : options) {
       optionMap.put(option.getOption(), option.getValue());
     }
+
```

Comment:
Could you add an example of using v4 in the javadoc (https://github.com/googleapis/google-cloud-java/blob/836d7d5860578ab5ac2c05a26e4386707dda7eb6/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java#L2121)?